src/RolloutScorer/RolloutScorer.Tests/RolloutScorerTests.cs

```diff
@@ -39,10 +39,9 @@ public static IEnumerable<object[]> GetUriTestCases()
         public void RolloutScorerTests_SetUp()
         {
             _rolloutScorer = new RolloutScorer();
-            Config config = Utilities.ParseConfig();
-            _rolloutScorer.RolloutWeightConfig = config.RolloutWeightConfig;
-            _rolloutScorer.RepoConfig = config.RepoConfigs.First();
-            _rolloutScorer.AzdoConfig = config.AzdoInstanceConfigs.Find(a => a.Name == _rolloutScorer.RepoConfig.AzdoInstance);
+            _rolloutScorer.RolloutWeightConfig = StandardConfig.DefaultConfig.RolloutWeightConfig;
```

Comment:
nit: initializations are cleaner looking this way: 

```
_rolloutScorer = new RolloutScorer 
{
    RolloutWeightConfig = StandardConfig.DefaultConfig.RolloutWeightConfig,
   ...
};
```

Reply:
they really do...i love doing them that way, too........................................................................................................,,,,,,,,,,,,,,,,,,,,,,,,,,